src/components/HelpResources/HelpResources.js

```diff
@@ -1,19 +1,132 @@
 import React from "react";
 import { GridContainer, Grid } from "@trussworks/react-uswds";
-import { HelpResourcesCards } from "../HelpResourcesCards/HelpResourcesCards";
+import { Link as RouterLink } from "react-router-dom";
+
+/*Import Icons from https://designsystem.digital.gov/components/icon/ using title case*/
+import {
+  Card,
+  CardHeader,
+  CardBody,
+  CardFooter,
+  IconHelpOutline,
+  IconReduceCapacity,
+  IconFlag,
+  // IconMap,
+  IconAlternateEmail,
+  CardGroup,
+} from "@trussworks/react-uswds";
 
 export const HelpResources = () => {
   return (
     <GridContainer>
       <Grid row gap>
         <Grid desktop={{ col: 8 }} className="text-left">
-          <h1 style={{ textAlign: "left" }}>Help & Resources</h1>
-          <p style={{ textAlign: "left" }}>
-            What to find in Help & Resources sections
-          </p>
+          <h1>Help & Resources</h1>
+          <p>What to find in Help & Resources sections</p>
         </Grid>
         <Grid row className="padding-2">
-          <HelpResourcesCards />
+          <aside>
```

Comment:
I'm thinking this should be an <article> tag since <aside> is generally used for "sidebars or call-out boxes" [according to MDN](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/aside). It really doesn't matter, but may impact how the page is interpreted via a screen reader?